server/server.js

```diff
@@ -1,13 +1,55 @@
-// make this a simple http server that writes to the response stream object
+/* make this a simple http server that writes to the response stream object
+ *
+ * the page should display the following message:
+ *
+ * Today is <day name>, <month name> <date>. Here are the available food trucks:
+ *
+ * list the food trucks returned by filterByDay, passing in the current day name.
+ * e.g. filterByDay(day); where "day" is determined using the enhancedDate module
+ * The list of trucks returned will be an array of food truck objects. Iterate 
+ * through the list, building up a string of food truck names. Once you're done 
+ * iterating through that list, display the string you built up.
+ *
+ * Remember that the response is a stream object that must be closed.
+*/
+ 
+var nodeDate = require('./enhancedDate'); //leave off file extension
+var myObject = nodeDate();
+myObject.setDate();
+var dayName = myObject.getDayName();
+var monthName = myObject.getMonthName();
+var date = myObject.now.getDay();
 
-// the page should display the following message:
+var trucks = require('./trucks');
+var truckObject = trucks();
+var dayTrucks = truckObject.filterByDay(dayName);
 
-// Today is <day name>, <month name> <date>. Here are the available food trucks:
+var express = require('express');  //require the express module which returns an object/function
+var app = express(); //creates and express application instance
+
+var pageContent = 'Today is ' + dayName + ', ' + monthName + ' ' + date + '. Here are the available food trucks:'
+
+//loop through the dayTrucks array and get .name for each item
+
+for(i=0;i<dayTrucks.length;i++){
+	pageContent = pageContent + '<br>'+ dayTrucks[i].name;	
+}
+
+//when this route handler fires
+
+app.get('/',function(request,response){
```

Comment:
This assignment should use the http module to create a server. It's very similar to express, but it's still different (you should use `http.createServer()` and `response.writeHead()` or `response.write()`, and `response.end()`.